Sources/YouboraPlugin.swift

```diff
@@ -37,11 +39,8 @@ public class YouboraPlugin: BasePlugin, AppStateObservable {
     private var ybPlugin: YBPlugin?
     
     /// The plugin's config
-    var config: AnalyticsConfig {
-        didSet {
-            self.addCustomProperties()
-        }
-    }
+    var config: AnalyticsConfig
+    
```

Comment:
The config is updated in three different functions.
(init, onUpdateMedia, onUpdateConfig)
I verified with the sample, playing first media and change media the values are different.

**But**, In case the update config is called after the media was prepared when the media is set to autoPlay and preload false, kalturaInfo will be empty.

Please fix it by calling addKalturaInfo after every update to the config or revert the change to call the function upon didSet. As well don't forget it in the init function cause in that case didSet is not called.